Add VOLTOCONFIG Env Var (#5752)
plone/volto: docs/source/configuration/environmentvariables.md

    yarn
    BUILD_DIR=dist node dist/server.js
    ```

`VOLTOCONFIG`

>     You can also provide a custom location for `volto.config.js` via an environment variable.

Reviewer: ```suggestion
    This environment variable allows you to specify a custom location for {file}`volto.config.js`.
```

    It can be relative to the current project or absolute.

    ```shell

>      VOLTOCONFIG=../../volto.config.js yarn start

Reviewer: ```suggestion
    VOLTOCONFIG=../../volto.config.js yarn start
```
